pkg/chartutil/requirements.go

```diff
@@ -216,6 +218,45 @@ func ProcessRequirementsTags(reqs *Requirements, cvals Values) {
 
 }
 
+func copyChartAsAlias(charts []*chart.Chart, dependentChart, aliasChart string) *chart.Chart {
+	var chartFound *chart.Chart
+	for _, existingChart := range charts {
+		if existingChart == nil {
+			continue
+		}
+		if existingChart.Metadata == nil {
+			continue
+		}
+		if existingChart.Metadata.Name != dependentChart {
+			continue
+		}
+
+		chartFound = new(chart.Chart)
+		chartFound.Metadata = &chart.Metadata{
```

Comment:
makes sense, however, earlier when i was trying something else for some other reason it did not work, I must be doing something crazy because i updated code and it works 👍